cli/src/commands/commit.rs

```diff
@@ -119,6 +120,11 @@ new working-copy commit.
                 commit.tree_id().clone(),
             )
             .write()?;
+
+        if !advanceable_branches.is_empty() {
```

Comment:
nit: emptiness check is redundant (though it's harmless.)

Reply:
Yeah... I added it intentionally to make it clearer to the reader that nothing happens if there's no branches to advance. Although I guess that might be obvious anyways. I'll delete and add a short comment.